recipes/openmesh-python/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "openmesh-python" %}
+{% set pypiname = "openmesh" %}
+{% set version = "1.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ pypiname[0] }}/{{ pypiname }}/{{ pypiname }}-{{ version }}.tar.gz
+  sha256: c1d24abc85b7b518fe619639f89750bf19ed3b8938fed4dd739a72f1e6f8b0f6
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - python
+    - pip
+    - setuptools_scm
+    - numpy
+    - pybind11
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+
+test:
+  imports:
+    - openmesh
+  source_files:
+    - tests
+  commands:
+    - cd tests
+    - python -m unittest discover
+
+about:
+  home: https://www.graphics.rwth-aachen.de:9000/OpenMesh/openmesh-python
+  license: MIT
```

I think this is wrong?
```suggestion
  license: BSD 3-Clause
```